src/index.d.ts

```diff
@@ -696,17 +785,49 @@ declare namespace admin.auth {
      */
     photoURL?: string | null;
 
+    /**
+     * The user's updated multi-factor related properties.
+     */
     multiFactor?: {
-      enrolledFactors: Array<{
-        uid?: string;
-        phoneNumber: string;
-        displayName?: string;
-        enrollmentTime?: string;
-        factorId: string;
-      }> | null;
+
+      /**
+       * The user's updated list of enrolled second factors. The provided list will always
```

Comment:
The updated list of enrolled second factors. The provided list overwrites the user's existing list of second factors. When null is passed, all of the user's existing second factors are removed.

Reply:
Done.